lnrpc/routerrpc/router.proto

```diff
@@ -990,12 +992,23 @@ message ForwardHtlcInterceptResponse {
     // For backwards-compatibility reasons, TEMPORARY_CHANNEL_FAILURE is the
     // default value for this field.
     lnrpc.Failure.FailureCode failure_code = 5;
+
+    // outgoing_amount_msat is used to set the outgoing htlc amount in a
+    // resumed HTLC.
+    uint64 outgoing_amount_msat = 6;
+
+    // custom_records is used to set the custom records in a resumed HTLC.
```

Comment:
I know you're probably still in the evaluation phase, but I think this should document what kind of custom records can be modified (e.g. custom TLV fields of the `update_add_htlc` p2p message).

Reply:
Yes, good point. Custom records on the `update_add_htlc` p2p message as opposed to the custom records in the onion.

Reply:
> Custom records on the update_add_htlc p2p message as opposed to the custom records in the onion.

Yep